lab3.py

```diff
@@ -0,0 +1,61 @@
+"""
+1. Znaleźć w internecie API ze źródłem danych o rynkach finansowych, przykłady:
+https://bittrex.github.io/api/v1-1
+https://bitbay.net/en/public-api
+https://www.tradingview.com/rest-api-spec/#section/Authentication
+
+Stworzyć prostą funkcję, która łączy się z danym API, pobiera listę ofert kupna oraz listę ofert sprzedaży i printuje do konsoli. (5pkt)
+
+2. Znaleźć API z drugiego źródła - giełdy / instytucji finansowej, wybrać jeden zasób finansowy, a właściwie ich parę (bitcoin-usd / ropa-usd / złoto-usd / eur - usd)
+porównać gdzie bardziej opłaca się kupić (oferty sprzedaży są niższe), a gdzie sprzedać (oferty kupna są wyższe) (5pkt)
+"""
+
+import requests
+
+
+def get_bitbay_orderbook_data():
+    response_request = requests.get(
+        'https://bitbay.net/API/Public/BTCPLN/orderbook.json')
+    return response_request.json()
+
+
+def get_bitbay_ticker_data():
+    response_request = requests.get(
+        'https://bitbay.net/API/Public/BTCPLN/ticker.json')
+    return response_request.json()
+
+
+def get_blockchain_ticker_data():
+    response_request = requests.get(
+        'https://blockchain.info/ticker')
+    return response_request.json()
+
+
+orderbook_data = get_bitbay_orderbook_data()
+sell_offers = orderbook_data['bids']
+buy_offers = orderbook_data['asks']
+
+print(f"Bids list: ")
+for offer in enumerate(sell_offers):
+    print(f"{offer[0]}. {offer[1][0]}zł")
+
+print(f"First 10 asks: ")
+for offer in enumerate(buy_offers[:10]):
+    print(f"{offer[0]}. {offer[1][0]}zł")
+
+bitbay_ticker = get_bitbay_ticker_data()
+blockchain_ticker = get_blockchain_ticker_data()
+
+bitbay_sell = bitbay_ticker['bid']
+bitbay_buy = bitbay_ticker['ask']
+
+blockchain_sell = blockchain_ticker['PLN']['sell']
+blockchain_buy = blockchain_ticker['PLN']['buy']
+
+print(
```

Comment:
staraj się strukturyzować swój kod, dzielić na segmenty, funkcje, ... U Ciebie wygląda to jak skrypcik, który wykonuje się z góry na dół. Aczkolwiek poza tą okolicą jest na prawdę ok. Fajnie, że wydzieliłaś np. pobieranie ofert w funkcje. Trzymaj się tego, a nie skryptowania;)